src/main/java/uk/gov/justice/hmpps/prison/api/model/AgencyPrisonerPayProfile.kt

```diff
@@ -24,7 +24,7 @@ data class AgencyPrisonerPayProfile(
   val payFrequency: Int?,
 
   @Schema(description = "The number of absences that are acceptable within one week", example = "5")
-  val weeklyAbsenceLimit: Int?,
+  val weeklyAbsenceLimit: Int? = null,
```

Comment:
I've checked the prod data and noticed that we also have nulls in minHalfDayRate, maxHalfDayRate, maxPieceWorkRate, maxBonusRate and defaultPayBandCode - would you mind making these nullable too while you're in here? Thanks.

Reply:
Sorted @mikehalmamoj